ClimbingGearBackend/Interfaces/IUserGearRepository.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ClimbingGearBackend.Models;
+using System.Security.Claims;
+
+namespace ClimbingGearBackend.Interfaces
+{
+  public interface IUserGearRepository
```

Comment:
This would be a good place to document what each of these methods are expected to do.